input_data.go

```diff
@@ -6,7 +6,9 @@ type Modifier int
 
 var (
 	Mouse mouse
+)
 
+const (
 	MOVE    = Action(0)
```

Comment:
Although not strictly a change of this pull request, where we are here.

Define aliases using the constants they refer to rather than the value. This also make it possible to remove the comments without loss of detail.

```diff
-MouseButtonLeft   MouseButton = 0 // equivalent for MouseButton1
-MouseButtonRight  MouseButton = 1 // equivalent for MouseButton2
-MouseButtonMiddle MouseButton = 2 // equivalent for MouseButton3
+MouseButtonLeft   MouseButton = MouseButton1
+MouseButtonRight  MouseButton = MouseButton2
+MouseButtonMiddle MouseButton = MouseButton3
```